examples/weighting_samples.rs

```diff
@@ -4,11 +4,11 @@ use streaming_iterator::*;
 use utils::*;
 
 fn wd_iterable_counter_demo() {
-    println!("\n\n -----WdIterable Counter Demo----- \n\n");
+    println!("\n\n -----Weight Counter Demo----- \n\n");
```

Comment:
I did not rename some of the Yaml/File themed Iterables, but in principle we [should](https://rust-lang.github.io/api-guidelines/naming.html#iterator-type-names-match-the-methods-that-produce-them-c-iter-ty). Want to do them? can push such to this PR. 
I use `sed 's/BlaIterable/Blargh/g' -i */*.rs`.